<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseBrowser</class>
 <widget class="QMainWindow" name="DatabaseBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DatabaseBrowser</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="ToolBarLayout"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="MemoryViewControl" name="MemoryVC" native="true"/>
      <widget class="HeadersViewControl" name="HeadersVC" native="true"/>
      <widget class="TablesViewControl" name="TablesVC" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MemoryViewControl</class>
   <extends>QWidget</extends>
   <header>Views/memoryviewcontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HeadersViewControl</class>
   <extends>QWidget</extends>
   <header>Views/headersviewcontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TablesViewControl</class>
   <extends>QWidget</extends>
   <header>Views/tablesviewcontrol.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
